emoji/src/main/java/com/vanniktech/emoji/EmojiPopup.java

```diff
@@ -83,11 +83,16 @@
     }
   };
 
-  EmojiPopup(@NonNull final View rootView, @NonNull final EmojiEditText emojiEditText,
-             @Nullable final RecentEmoji recent, @Nullable final VariantEmoji variant) {
+  /**
+   * Constructor.
+   */
+  EmojiPopup(@NonNull final View rootView
+            , @NonNull final EmojiEditTextInterface editInterface
+            , @Nullable final RecentEmoji recent
+            , @Nullable final VariantEmoji variant) {
     this.context = Utils.asActivity(rootView.getContext());
     this.rootView = rootView.getRootView();
-    this.emojiEditText = emojiEditText;
+    this.editInterface = editInterface;
```

Comment:
we should throw an IllegalArgumentException here in case the `EditInterface` is not casteable to a `View`